include/ck/tensor_description/tensor_descriptor.hpp

```diff
@@ -213,6 +216,9 @@ struct TensorDescriptor
     Transforms transforms_;
     ElementSize element_size_;
     ElementSpaceSize element_space_size_;
+
+    private:
+    std::size_t real_size_;
```

Comment:
what is ```real_size```

also ```std``` should not be used in CK core code, use ```long_index_t``` instead
https://github.com/ROCmSoftwarePlatform/composable_kernel/wiki/Coding-Style